docs/kyma-environment-broker/08-07-swagger.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Check API using swagger
+type: Tutorials
+---
+
+This tutorial shows how to check the API schema using Swagger.
+
+The Swagger is injected into the KEB's container which exposes the Swagger UI on root endpoint.
+
+The Swagger UI static files are copied from the [official source](https://github.com/swagger-api/swagger-ui/tree/master/dist).
+
+The KEB uses swagger schema file mounted as volume to the Pod. You can find that schema [here](https://github.com/kyma-project/control-plane/blob/master/resources/kcp/charts/kyma-environment-broker/files/swagger.yaml).
```

Comment:
These sentences seem to be a bit out of context 🤔 Let's provide some general context to them.